lib/LaTeXML/Package/TeX.pool.ltxml

```diff
@@ -713,9 +713,9 @@ sub compare {
       "Expected a relational token for comparision", "Got " . Stringify($rel));
     return; } }
 
-DefConditional('\ifnum Number Token Number',       sub { compare($_[1], $_[2], $_[3]); });
-DefConditional('\ifdim Dimension Token Dimension', sub { compare($_[1], $_[2], $_[3]); });
-DefConditional('\ifodd Number',                    sub { $_[1]->valueOf % 2; });
+DefConditional('\ifnum Number Token Number SkipSpaces',       sub { compare($_[1], $_[2], $_[3]); });
+DefConditional('\ifdim Dimension Token Dimension SkipSpaces', sub { compare($_[1], $_[2], $_[3]); });
+DefConditional('\ifodd Number SkipSpaces',                    sub { $_[1]->valueOf % 2; });
```

Comment:
Something I would like to understand better is whether this behavior of skipping spaces after the conditional test is something that should be done for *all* conditional types.

In other words, while patching `\ifnum` was all it took to get the etoolbox machinery to work, I wonder if it isn't better to patch `DefConditional` itself centrally, so that you can never declare a conditional that doesn't skip spaces after its test.

Would like to read up on this in the texbook possibly... Just a note for now.